registry/remote/interface_test/interface_test.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright The ORAS Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package interface_test
```

Comment:
We can put this file under the `registry/remote` directory, and call it `package remote_test`. Just like what we did for [`example_test.go`](https://github.com/oras-project/oras-go/blob/main/registry/remote/example_test.go#L15).

Reply:
Changed accordingly.